intel_extension_for_transformers/neural_chat/models/model_utils.py

```diff
@@ -1437,15 +1437,15 @@ def predict(**params):
     else:
         output = tokenizer.decode(generation_output.sequences[0], skip_special_tokens=True)
     if "### Response:" in output:
-        return output.split("### Response:")[1].strip()
```

Comment:
do we have a test to cover this? 

suggest not using a magic number directly, but you can define a meaningful variable name first and refer to it.

Reply:
UT added to cover this.
The magic number is replaced by the variable `identifier_flag`.